jetty-ee11/jetty-ee11-servlet/src/main/java/org/eclipse/jetty/ee11/servlet/ServletApiResponse.java

```diff
@@ -632,4 +656,132 @@ public String toString()
             return HttpCookie.toString(this);
         }
     }
+
+    public static class CrossContextInclude extends ServletApiResponse
+    {
+        public CrossContextInclude(ServletContextResponse servletContextResponse)
+        {
+            super(servletContextResponse);
+        }
+
+        @Override
+        public void setCharacterEncoding(String charset)
+        {
+            // NOOP for include.
```

Comment:
The javadoc for this method doesn't mention it isn't valid for an include. Are you sure about this?

Reply:
See the link to RequestDispatcher#include added above.   An include cannot modify response headers, so all the methods that can do that are noops.